b0tweb/MessageHandlers/CommandMessageHandler.cs

```diff
@@ -5,10 +5,27 @@ namespace b0tweb.MessageHandlers
 {
     class CommandMessageHandler : AbstractMessageHandler
     {
+        /// <summary>
+        /// The nickname of the current IRC bot user.
+        /// </summary>
+        private string _nick;
+
+        public CommandMessageHandler(string nick)
+        {
+            this._nick = nick;
+        }
+
         protected override void BuildHandler(object sender, IrcEventArgs e)
         {
-            string response = CommandRegistry.ExecuteSync(e.Data.Message);
+            string[] parts = e.Data.Message.Split(' ');
 
+            //match *, nick {message} and nick: {message}
+            if (parts.Length < 2 || !(parts[0] == "*" || parts[0] == this._nick || parts[0] == this._nick + ":"))
```

Comment:
also have to add it to kill switch handler

Reply:
Has been updated